rs/state_layout/src/state_layout.rs

```diff
@@ -71,6 +71,8 @@ pub struct RwPolicy<'a, Owner> {
     lifetime_tag: PhantomData<&'a Owner>,
 }
 
+pub enum Verification {}
```

Comment:
Should we maybe call this `InVerification`? Also I think a doc comment would be useful what this access policy is supposed to provide.